CONTRIBUTING.md

```diff
@@ -36,24 +42,32 @@ Optionally you can install the [EditorConfig](https://marketplace.visualstudio.c
         <td align="center">AzureADWindowsAuthenticator</td>
     </tr>
     <tr>
-        <th rowspan="3">Modules</td>
-        <td align="center" colspan="2">Commands.Profile</td>
+        <td align="center" colspan="2">Common.Api</td>
+    </tr>
+    <tr>
+        <td align="center" colspan="2">Common.Client</td>
+    </tr>
+    <tr>
+        <th rowspan="4">Modules</td>
+        <td align="center" colspan="3">Commands.Profile</td>
     </tr>
     <tr>
         <td align="center">Commands.Workspaces</td>
         <td align="center">Commands.Reports</td>
+        <td align="center">Commands.Data</td>
     </tr>
 <table>
 
-> * Test Projects are located next to implementation.
-> * AzureADWindowsAuthenticator has an independent depenency stack as it's compiled against .NET 4.6 (opposed to .NET Core) to be a bridge for ADAL on Windows (providing interactive login).
+> * Test Projects are located next to implementation (module project).
+> * AzureADWindowsAuthenticator has an independent dependency stack as it's compiled against .NET 4.6 (opposed to .NET Core) to be a bridge for ADAL on Windows (providing interactive login).
```

Comment:
"opposed to .NET Core" -> "as opposed to .NET Core"
"to be a bridge" -> "to act as a bridge"

Reply:
Ok